### DISCUSSION TITLE
ValueError with cov-bsv

### DISCUSSION THREAD

**Discussion by greybraun** [explosion/spaCy#11479] [Help: Other Questions]
I wanted to load model on this site.
https://spacy.io/universe/project/cov-bsv

simple code:
import cov_bsv # model
nlp = cov_bsv.load() # load model

error you have seen occoured on second line.

**Comment by polm** [comment]
Let me link to #7743 where this started for context...

The `cov-bsv` project seems to only support spaCy v2, so if you want to use it the easiest thing woud be to use a v2 version of spaCy. 

Also, you asked in the other thread, but we don't take live calls for support here.

**Comment by greybraun** [comment]
Thanks for your good answer.
